docker-compose.test.yml

```diff
@@ -38,3 +46,17 @@ services:
     ports:
       - '8080:8080'
     restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://${LOCAL_HOST}:${MOCKOON_PORT}"]
+
```

Comment:
Please remove the new line there and be consistent in the code sttyle through all docker compose containers